sdk/keyvault/keyvault/test/client_key_POC.c

```diff
@@ -13,6 +16,39 @@
 
 int exit_code = 0;
 
+az_span get_key_version(az_http_response * response) {
+  az_http_response_parser parser = { 0 };
+  az_span body = { 0 };
+  az_result r = az_http_response_parser_init(&parser, az_span_builder_result(&response->builder));
```

Comment:
We should use `AZ_RETURN_IF_FAILED` instead of `r`.

Reply:
this is POC (kind of testing). Simulating what a customer would do/use with our SDK.